config/webhook/manager_webhook_patch.yaml

```
@@ -11,6 +11,7 @@ spec:
           args:
             - "--metrics-addr=127.0.0.1:8080"
             - "--webhook-port=9443"
+            - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},AKS=${EXP_AKS:=false},ClusterResourceSet=${EXP_CLUSTER_RESOURSE_SET:=false}"
```

```suggestion
            - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},AKS=${EXP_AKS:=false},ClusterResourceSet=${EXP_CLUSTER_RESOURCE_SET:=false}"
```

(it was a typo in CAPI that was fixed subsequently)